### PR TITLE
Misc map issues

### LINKED ISSUE #198: Cutscene bug ?-?
it only cuts when u mark it undone

### LINKED ISSUE #208: Auto-close cutscene after animation is done

### LINKED ISSUE #224: Camera face character

### LINKED ISSUE #241: taking a dip
image

### LINKED ISSUE #243: Update map for desktop view
image

### LINKED ISSUE #244: Memento cut into each other abit
Screenshot 2021-11-06 at 3 53 22 PM
the colours are so washed out :-( sad..

### LINKED ISSUE #257: Update map initial zoom on desktop view
It's too zoomed out

### PR DESCRIPTION
**Pull request by jialin7878**
Closes #224 
Closes #241 
Closes #208 
Closes #244 
Closes #243 
Closes #198 
Closes #257

### CODE REVIEW

File: frontend/src/pages/home/HomePage.tsx [hunk 1]
```diff
@@ -107,6 +108,9 @@ const HomePage: React.FC = () => {
       openChallengeName: openChallengeName,
     });
   };
+  useEffect(() => {
+    dispatch(loadAllOngoingChallengeMaps());
+  }, []);
```
Comment (sebastiantoh) [inline frontend/src/pages/home/HomePage.tsx:113]: i think lines `dispatch(loadAllFriendRequests());` should be shifted here as well. Could you help to make the change if you agree?

**Review by sebastiantoh — APPROVED** [review]
looks good, ty!

File: frontend/src/pages/home/HomePage.tsx (hunk 1 above)
Comment (jialin7878) [reply frontend/src/pages/home/HomePage.tsx]: agree, shifted!

**Comment by sebastiantoh** [comment]
merge conflict :(

**Comment by jialin7878** [comment]
omg that was satisfying

**Pull request merged**